component/http/cache/cache_model.go

```diff
@@ -0,0 +1,68 @@
+package cache
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// handlerRequest is the dedicated request object for the cache handler
+type handlerRequest struct {
+	header string
+	path   string
+	query  string
+}
+
+// toCacheHandlerRequest transforms the http Request object to the cache handler request
+func toCacheHandlerRequest(req *http.Request) *handlerRequest {
```

Comment:
since we are in the cache package could we name if just:
```suggestion
func toHandlerRequest(req *http.Request) *handlerRequest {
```